### PR TITLE
Allow using the admin module

### PR DESCRIPTION
**Pull request by Gjum**

### CODE REVIEW

File: users/models.py [hunk 1]
```diff
@@ -32,6 +32,28 @@ def create_user(self, email, username, password):
...
+            raise ValueError('Superusers must have an email address')
+
+        if not username:
+            raise ValueError('Superusers must have an username')
+
+        if not password:
+            raise ValueError('Superusers must have a password')
+
+        superuser = self.model(
+            email=self.normalize_email(email),
```
Comment (metalmatze) [inline users/models.py:46]: Did you run the `format code` with pyCharm?

Comment (Gjum) [inline users/models.py:46]: Yup. Why?

Comment (metalmatze) [inline users/models.py:46]: `email=self` there no spaces before and after the equals signs

Comment (Gjum) [inline users/models.py:46]: I copied that code from you, so I assumed it's the format we use.
Also, according to PEP you do not use spaces around `=`s in kwargs.

**Pull request merged**